.classpath

```diff
@@ -2,7 +2,32 @@
 <classpath>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="src" path="Test"/>
+<<<<<<< HEAD
```

Comment:
It looks like you left the conflicted section in here.  Please remove the conflict markers and the libraries that were added back in.